File: Android_Nativo_Java - Novo/app/src/main/java/com/example/gpos700x/MainActivity.java
package com.example.gpos700x;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Build;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ListView;
import android.widget.TextView;

import com.example.gpos700x.ExemploCodigoBarras1.CodigoBarras1;
import com.example.gpos700x.ExemploCodigoBarras2.CodigoBarras2;
import com.example.gpos700x.ExemploImpressora.Impressora;
import com.example.gpos700x.ExemploMifare.ExemploMifare;
import com.example.gpos700x.ExemploTEF.Tef;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    public static final String G700X = "GPOS700X";
    private static final String version = "v1.0.0";

    public static String Model = Build.MODEL;

    ArrayList<Projeto> projetos = new ArrayList<Projeto>();
    ListView lvProjetos;

    TextView txtProject;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        lvProjetos = findViewById(R.id.lvProjetos);

        projetos.add(new Projeto("Código de Barras", R.drawable.barcode));
        projetos.add(new Projeto("Código de Barras V2",R.drawable.qr_code));
        projetos.add(new Projeto("Impressão",R.drawable.print));


        if(Model.equals(G700X)){
            projetos.add(new Projeto("Mifare",R.drawable.nfc));
//            projetos.add(new Projeto("NFC Id",R.drawable.nfc1));

        }
//        else{
//            projetos.add(new Projeto("NFC Leitura/Gravação",R.drawable.nfc2));
//        }

        projetos.add(new Projeto("TEF", R.drawable.tef));

        ProjetoAdapter adapter = new ProjetoAdapter(getBaseContext(), R.layout.listprojetos, projetos);
        lvProjetos.setAdapter(adapter);
        lvProjetos.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
                Projeto projeto = (Projeto) lvProjetos.getItemAtPosition(i);

                Intent intent = null;
                switch (projeto.getNome()){
                    case "Código de Barras":
                        // Toast.makeText(getApplicationContext(), projeto.getNome(), Toast.LENGTH_LONG).show();
                        intent = new Intent(MainActivity.this, CodigoBarras1.class);
                        break;
                    case "Código de Barras V2":
                        // Toast.makeText(getApplicationContext(), projeto.getNome(), Toast.LENGTH_LONG).show();
                        intent = new Intent(MainActivity.this, CodigoBarras2.class);
                        break;
                    case "Impressão":
                        // Toast.makeText(getApplicationContext(), projeto.getNome(), Toast.LENGTH_LONG).show();
                        intent = new Intent(MainActivity.this, Impressora.class);
                        break;
                    case "Mifare":
                        intent = new Intent(MainActivity.this, ExemploMifare.class);
                        break;
                    case "TEF":
                        intent = new Intent(MainActivity.this, Tef.class);
                        break;
                }
                if(intent != null){
                    startActivity(intent);
                }
            }
        });
    }
}

File: Kotlin/app/src/main/java/com/example/gpos700x_kot/MainActivity.kt
package com.example.gpos700x_kot

import android.content.Intent
import android.os.Build
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ListView
import android.widget.TextView
import com.example.gpos700x_kot.ExemploCodigoBarras1.CodigoBarras1
import com.example.gpos700x_kot.ExemploCodigoBarras2.CodigoBarras2
import com.example.gpos700x_kot.ExemploImpressora.Impressora
import com.example.gpos700x_kot.ExemploMifare.ExemploMifare
import com.example.gpos700x_kot.ExemploTEF.Tef

class MainActivity : AppCompatActivity() {


    companion object {
        const val G700X = "GPOS700X"
        private const val version = "v1.0.0"
        var Model = Build.MODEL
    }

    private val projetos = ArrayList<Projeto>()
    private lateinit var lvProjetos: ListView
    private lateinit var txtProject: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        lvProjetos = findViewById(R.id.lvProjetos)

        projetos.add(Projeto("Código de Barras", R.drawable.barcode))
        projetos.add(Projeto("Código de Barras V2", R.drawable.qr_code))
        projetos.add(Projeto("Impressão", R.drawable.print))

        if (Model == G700X) {
            projetos.add(Projeto("Mifare", R.drawable.nfc))
        }

        projetos.add(Projeto("TEF", R.drawable.tef))

        val adapter = ProjetoAdapter(applicationContext, R.layout.listprojetos, projetos)
        lvProjetos.adapter = adapter
        lvProjetos.setOnItemClickListener { _, view, i, _ ->
            val projeto = lvProjetos.getItemAtPosition(i) as Projeto

            var intent: Intent? = null
            when (projeto.nome) {
                "Código de Barras" -> intent = Intent(this@MainActivity, CodigoBarras1::class.java)
                "Código de Barras V2" -> intent = Intent(this@MainActivity, CodigoBarras2::class.java)
                "Impressão" -> intent = Intent(this@MainActivity, Impressora::class.java)
                "Mifare" -> intent = Intent(this@MainActivity, ExemploMifare::class.java)
                "TEF" -> intent = Intent(this@MainActivity, Tef::class.java)
            }
            if (intent != null) {
                startActivity(intent)
            }
        }
    }
}